addons/stock/models/stock_move.py

```diff
@@ -610,16 +592,8 @@ def _delay_alert_check(self, new_date=None):
                 next_nondone_moves.write({'delay_alert_date': False})
```

Comment:
next_nondone_moves is emtpy now. Should be fixed or removed.

Reply:
The delay alert date is clean futher by 0f6f78db473d7c8096d381ee52ccff196e147bc4